js/components/App.js

```diff
@@ -24,6 +38,8 @@ export default Relay.createContainer(App, {
             node {
               id,
               name,
+              enabled,
+              ${ToggleWidgetEnableMutation.getFragment('widget')},
```

Comment:
We might be able to make this error message less confusing. The idea here is that your component might happen to query for fields needed by the mutation by mere luck, but that is not very robust because somebody may come along later and make an unrelated change, unaware of the mutation, which removes a necessary fields. So, we want you to compose the mutation fragment explicitly.